core/src/avm2/object/xml_object.rs

```diff
@@ -196,6 +196,29 @@ impl<'gc> TObject<'gc> for XmlObject<'gc> {
             }
         }
 
+        let multiname = if !name.has_explicit_namespace() {
```

Comment:
I doubt this is really correct. I think it's likely that this could modify Multinames that are already names used for XML and we would e.g. strip two leading @@.

Reply:
I think this could be fixed by changing the check to `!name.has_explicit_namespace() && !name.is_attribute()`